### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by ghost** [abraunegg/onedrive#1506] [Q&A] [answered]
Does onedrive "autostart" by default or I always have to run the sync command every time?

Can I set monitor mode in config file somehow, to be applied by default? So instead of `onedrive --synchronize -v --monitor` I can just simply write `onedrive --synchronize` and monitor mode will be enabled when starting.

What happens if the command line process stops somehow? If I run sync again does it check for already installed folders do does it overwrites everything from the beginning?

How automated is the default config set up? How much do I have to interact with a terminal if I want to sync auto?

If I synced a folder and I add it to the do not sync dir, does it disappear locally if a sync is initiated or does it have to be re-synced?

Can a process run if there are no terminal? For example do I always have to have a terminal open to control the process even in monitor mode?

Can I change the download speed while syncing?

**Comment by abraunegg** [comment, marked as answer]
@Hogerys 
> Does onedrive "autostart" by default or I always have to run the sync command every time?

This depends. By default, should be 'no', however, if you are using Ubuntu or Debian, you are hit with this issue - https://github.com/abraunegg/onedrive/issues/1274

> Can I set monitor mode in config file somehow, to be applied by default? So instead of onedrive --synchronize -v --monitor I can just simply write onedrive --synchronize and monitor mode will be enabled when starting.

You cannot run `--synchronize` & `--monitor` together. It is one or the other. Please read the documentation.

> What happens if the command line process stops somehow? If I run sync again does it check for already installed folders do does it overwrites everything from the beginning?

If a sync stops, the client will validate existing data (no overwrite unless data has changed somewhere) before dealing with new content. Suggest you read the documentation and use `--dry-run` in your testing - this is why it exists.

> How automated is the default config set up? How much do I have to interact with a terminal if I want to sync auto?

Please read the documentation:
* https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#running-onedrive-in-monitor-mode
* https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#running-onedrive-as-a-system-service

> If I synced a folder and I add it to the do not sync dir, does it disappear locally if a sync is initiated or does it have to be re-synced?

No, only new content will not be synced. Suggest you read the documentation and use `--dry-run` in your testing - this is why it exists.

> Can a process run if there are no terminal? For example do I always have to have a terminal open to control the process even in monitor mode?

Suggest you read the documentation on running as a service

> Can I change the download speed while syncing?

Suggest you read the documentation re rate limiting
